plugins-compat-tester/pom.xml

```diff
@@ -63,59 +63,64 @@
 	  <artifactId>maven-model</artifactId>
 	  <version>3.0.2</version>
 	</dependency>
-  	
 
+
+    <dependency>
+      <groupId>org.apache.maven.scm</groupId>
+      <artifactId>maven-scm-api</artifactId>
+      <version>${maven.scm.version}</version>
+    </dependency>
     <!--TODO(oleg_nenashev): Remove all providers except Git? -->
     <!-- scm providers declaration -->
     <dependency>
       <groupId>org.apache.maven.scm</groupId>
       <artifactId>maven-scm-provider-accurev</artifactId>
```

Comment:
!!

Reply:
yup. look at `SCMManagerFactory` it's using `maven-scm-api` packages directly so it MUST be declared as it is used and not rely on lucky/hazardous transitivity..
AFAIK we have sources only in git so we would remove every other scm-providers

Reply:
At some point we should consider just deleting the whole dep and directly calling `git` from `ProcessBuilder`.